api/metrics_service.proto

```diff
@@ -37,9 +37,9 @@ message StreamMetricsMessage {
   repeated io.prometheus.client.MetricFamily envoy_metrics = 2;
 }
 
-// Configuration structure.
+// Configuration structure for Metrics Service. It is a :ref: 'StatsSink <envoy_api_msg_StatsSink>'
```

Comment:
I think these need to be backticks. Can you temporarily remove draft status and verify locally that the doc links render as expected?